samples/tab-personal-sso-quickstart/csharp_dotnetcore/wwwroot/js/auth.js

```diff
@@ -1,8 +1,8 @@
 ﻿let accessToken;
 
 $(document).ready(function () {
-    microsoftTeams.initialize();
-   
+    microsoftTeams.app.initialize();
```

Comment:
Assuming this is an imported namespace referring to the teamsjs library like in the js sample? Maybe consider using a generic term like host instead of it being microsoftTeams specific.

Reply:
For js sample, we were using name imports to directly use the namespace, which was not possible since we are using cdn instead of npm package, so had to import them using parent namespace